vite.lib.config.ts

```diff
@@ -13,7 +13,7 @@ export default defineConfig({
 
     lib: {
       entry: "src/main.ts",
-      formats: ["es"],
+      formats: ["es", "cjs"],
```

Comment:
Just in case someone decides to use in a Webpack-based app (AFAIK, Webpack still doesn't support ES Modules up to this day :confused:).